lamby/database/stub.py

```diff
@@ -0,0 +1,67 @@
+"""
```

Comment:
Ok, this is pretty cool.

Reply:
We can just keep it internal for now because it doesn't add any onnx files.  Just dummy data that i used to test sorting

Reply:
Ok cool.